pycine/file.py

```diff
@@ -45,6 +48,67 @@ def read_chd_header(chd_file):
 
     return header
 
+def taggedBlock(cine_file, header):
+    #with help of https://raw.githubusercontent.com/soft-matter/pims/master/pims/cine.py
+    with open(cine_file, "rb") as f:
+        header_length = ct.sizeof(header['cinefileheader'])
+        bitmapinfo_length = ct.sizeof(header['bitmapinfoheader'])
+        if not header['cinefileheader'].OffSetup + header['setup'].Length < header["cinefileheader"].OffImageOffsets:
+            print('no tagged block available')
```

Comment:
As pycine is used as a library, it should not print anything.

Reply:
ok, no problem.